src/admin/admin.controller.ts

```diff
@@ -0,0 +1,25 @@
+import { Body, Controller, HttpCode, HttpStatus, Post, UploadedFile, UseInterceptors } from '@nestjs/common';
+import { FileInterceptor } from '@nestjs/platform-express';
+import { Express } from 'express';
+import { UploadStudentDataService } from '../student/upload-student-data.service';
+import { ApiResponse, CreateResponse } from '@Types';
+import { HrDto } from '../hr/dto';
+import { HrService } from '../hr/hr.service';
+
+@Controller('admin')
+export class AdminController {
+  constructor(private uploadStudentDataService: UploadStudentDataService, private hrService: HrService) {}
+
+  @Post('upload/file')
+  @HttpCode(HttpStatus.OK)
```

Comment:
Czy to niezbędne? Automatycznie na POST jest zawsze 201, na innych 200 (właśnie OK) - nie trzeba tego pisać.